docs/src/howto.md

```diff
@@ -212,9 +220,50 @@ The live viewer can be called with adding `; liveview=true` to the [`render`](@r
 
 For longer videos, it can happen that rendering takes some time.
 A long time.
-One way to reduce rendering time is that you can render a scaled version of the animation to check if everything is animated as expected. 
+One way to reduce rendering time is that you can render a scaled version of the animation to check if everything is animated as expected.
 
 By using `render(video; pathname="how_to.gif", rescale_factor=0.5)`, the rendering process can be sped up generally a factor of 2.
-This scales the frames of an animation down to half where a `Video(1000, 1000)` will be shown as a `Video(500, 500)` rendered video. 
+This scales the frames of an animation down to half where a `Video(1000, 1000)` will be shown as a `Video(500, 500)` rendered video.
 
 > **Note:** You might want to experiment with rendering to `mp4` instead of `gif` as well.
+
+## Why does `pos` always throws error?
+
+Both `get_position` and `pos` are a bit unintuitive in the sense that one may think that this should work:
+
+```julia
+vid = Video(500, 500)
+Background(1:50, (args...)->background("black"))
+o1 = Object(JCircle(O, 10, action=:fill))
+render(vid)
```

Comment:
I think you've missed a `o2` object here or something. Also maybe set the background color to white as otherwise it's a black circle on a black background